python/PyQt6/core/__init__.py.in

```diff
@@ -540,6 +540,7 @@ QgsException.__doc__ = "Defines a QGIS exception class."
 
 try:
    import numpy as _numpy
+   import shapely as _shapely
```

Comment:
shapely related things should be put in another try block, so that the raster part can work only if numpy is available